introduction.md

```diff
@@ -1,4 +1,8 @@
 Ehilà come va
 Hi there how are you doing
 
-My favourite Code School Path is Javascript
\ No newline at end of file
+My favourite Code School Path is Javascript
+
+* JS
+* Git
+* stuff
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?